src/Public/Responses.ps1

```diff
@@ -63,9 +63,15 @@ function Set-PodeResponseAttachment {
 
     # only attach files from public/static-route directories when path is relative
     $route = (Find-PodeStaticRoute -Path $Path -CheckPublic -EndpointName $EndpointName)
+    if ($route) {
+        $_path = $route.Content.Source
 
+    }
+    else {
+        $_path = Get-PodeRelativePath -Path $Path -JoinRoot
+    } 
```

Comment:
This breaks the core logic of this function, as it's only suppose to look in public/static folders. If we want the option to look at non-public/static folder then I'd recommend an `-AllowExternalFiles` switch to toggle this logic.

Reply:
let me compare it  with the 2.9 version.
it was not my intention to change the logic of this function

Reply:
```powershell

    # only attach files from public/static-route directories when path is relative
    $_path = (Find-PodeStaticRoute -Path $Path -CheckPublic -EndpointName $EndpointName).Content.Source

    # if there's no path, check the original path (in case it's literal/relative)
    if (!(Test-PodePath $_path -NoStatus)) {
        $Path = Get-PodeRelativePath -Path $Path -JoinRoot

        if (Test-PodePath $Path -NoStatus) {
            $_path = $Path
        }
    }
```

this is the original code. Probably I'm missing something but I don't see differences in the logic of the function

Reply:
reviewing this code I can see that 
```powershell
 else {
        $_path = Get-PodeRelativePath -Path $Path -JoinRoot
    }
``` 
is redundant. `Write-PodeAttachmentResponseInternal` first instruction is exactly the same. I can remove these 3 lines

Reply:
solved in #1266